docs/Reference/Pantheon-CLI-Subcommands.md

```diff
@@ -17,7 +17,21 @@ pantheon blocks import --from=<block-file>
 pantheon blocks import --from=/home/me/me_project/mainnet.blocks
 ```
 
-Imports blocks from the specified file into the blockchain database
+Imports blocks from the specified file into the blockchain database.
+
+### export
+
+```bash tab="Syntax"
+pantheon blocks export [--start-block=<LONG>] [--end-block=<LONG>] --to=<block-file>
+```
+
+```bash tab="Example"
+pantheon --network=rinkeby --data-path=/home/data/ blocks export --start-block=100 --end-block=300 --to=/home/exportblock.bin
+```
+
+Exports a block, or list of blocks from storage to a file in RLP format. 
```

Comment:
It might be worth explaining what happens if `--start-block` or `--end-block` are omitted:  start block will default to 0, and end block will default to the end of the chain.

Reply:
Good idea, thanks.